content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -5,38 +5,35 @@ weight: 60
 keywords: [kubernetes,multicluster]
 ---
 
-Instructions for the installation of Istio multicluster.
+An **Istio multicluster** enables Kubernetes control planes running a remote
+configuration to connect to **one** Istio control plane. Once you connect one
```

Comment:
I don't like the terminology "control planes running a remote configuration".  The context of configuration is not clear.   Could you say "disparate Kubernetes control planes to connect" or "distinct Kubernetes control planes to connect" or even "separate"

Reply:
@john-a-joyce Would this wording be better?
An **Istio multicluster** enables different Kubernetes control planes running a remote
configuration to connect to **one** Istio control plane.

Reply:
That wording is better but I really don't see the need the term "remote configuration" in this sentence at all.   If you think it is useful go ahead and keep your latest proposed wording.

Reply:
I think it is important since it signals that each control plane can run its own "remote configuration". I'll change the wording then. Thanks

Reply:
The wording here has changed completely thanks to your PR @john-a-joyce